src/backend/allocator/format.rs

```diff
@@ -82,6 +83,18 @@ macro_rules! format_tables {
             }
         }
 
+        /// Returns the depth of the specified format.
```

Comment:
what makes a depth different from the bpp?

Reply:
Also let's mention this in the module docs

Reply:
It is the amount of used bits per pixel. So any padding or non-alpha ("X") parts of the format are not counted towards the depth, but the bpp.

Older drm apis require that information to create a framebuffer.

Reply:
Added to module docs